### PR TITLE
Checkout page

### PR DESCRIPTION
**Pull request by tkevans**
Added some UI for checkout page and confirm page. I switch to use TableViewController for checkout page. However, I have troubles connecting ProfileButton and ConfirmButton wth the code (from CheckViewController.swift

### CODE REVIEW

**Comment by tkevans** [comment]
Still have troubles connecting all buttons from the CheckViewController.swift to the actual code. It shows object can only be "Exit".

screen shot 2018-04-22 at 3 10 13 pm

**Comment by wenjunzhu1991** [comment]
add a new button, and create a segue to checkViewController

**Comment by wenjunzhu1991** [comment]
image
like this

**Comment by tkevans** [comment]
@wenjunzhu1991 I know how to make a segue. However, the button can go two directions in this case, for example when user click "confirm" button to confirm order, if he or she has logged in then the button will lead to "order successfully", otherwise the button will lead to "login" page. To make such decisions we have to use code to perform it I think.

**Comment by HackBL** [comment]
After you drag the button into corresponding viewController, it'll generate a button func, and inside the func, you could do some conditions to determine where next step will be when the button clicked.

**Comment by HackBL** [comment]
Also, if do this way drag the button into viewController, you cannot drag the button into other viewControllers in storeboard.

**Comment by tkevans** [comment]
Hey guys, I have changed the checkout page, fixed all the bugs. Now scenes switch nicely. Go take a look!

**Comment by HackBL** [comment]
Great job, you could merge it!

File: FoodHaus_App/project-swift/Controller/Register/LoginViewController.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ class LoginViewController: UIViewController {
         
         Auth.auth().addStateDidChangeListener() { auth, user in
             if user != nil {
-                self.performSegue(withIdentifier: "LoginToHome", sender: nil)
+                self.performSegue(withIdentifier: "LoginToCheck", sender: nil)
```
Comment (HackBL) [inline FoodHaus_App/project-swift/Controller/Register/LoginViewController.swift:15]: This is correct direction to go to the Check page when click Login

File: FoodHaus_App/project-swift/Model/Menu/CheckTableModel.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+//  CheckTableModel.swift
+//  project-swift
+//
+//  Created by Tyler Evans on 4/22/18.
+//  Copyright © 2018 Bo Li. All rights reserved.
+//
+
+import UIKit
+
+class CheckTableModel
```
Comment (HackBL) [inline FoodHaus_App/project-swift/Model/Menu/CheckTableModel.swift:11]: Seems you next step is doing the job like pass the data from menu into the check model, and then present all data into viewController

**Pull request merged**